ui/app/components/clients/page/counts.hbs

```diff
@@ -151,7 +151,7 @@
         </Hds::Alert>
       {{/if}}
 
-      <Clients::Counts::NavBar @showSecretsSync={{this.showSecretsSync}} />
+      <Clients::Counts::NavBar @showSecretsSyncClientCounts={{(or @hasSecretsSyncClients this.flags.showSecretsSync)}} />
```

Comment:
```suggestion
      <Clients::Counts::NavBar @showSecretsSyncClientCounts={{(or this.hasSecretsSyncClients this.flags.showSecretsSync)}} />
```
^ to ensure we're using the `get`ter instead of a passed down arg 🙂